examples/How_to_call_functions_with_chat_models.ipynb

```diff
@@ -652,103 +732,130 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
    "id": "65585e74",
-   "metadata": {},
-   "outputs": [],
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-05-14T21:53:53.215856Z",
+     "start_time": "2024-05-14T21:53:53.213850Z"
+    }
+   },
    "source": [
     "def ask_database(conn, query):\n",
     "    \"\"\"Function to query SQLite database with a provided SQL query.\"\"\"\n",
     "    try:\n",
     "        results = str(conn.execute(query).fetchall())\n",
     "    except Exception as e:\n",
     "        results = f\"query failed with error: {e}\"\n",
-    "    return results\n",
-    "\n",
-    "def execute_function_call(message):\n",
-    "    if message.tool_calls[0].function.name == \"ask_database\":\n",
-    "        query = json.loads(message.tool_calls[0].function.arguments)[\"query\"]\n",
-    "        results = ask_database(conn, query)\n",
-    "    else:\n",
-    "        results = f\"Error: function {message.tool_calls[0].function.name} does not exist\"\n",
     "    return results"
-   ]
+   ],
+   "outputs": [],
+   "execution_count": 81
   },
   {
-   "cell_type": "code",
-   "execution_count": 19,
-   "id": "38c55083",
    "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "##### Steps to invoke a function call using Chat Completions API: \n",
```

Comment:
Nit but could you make this a little bit more readable? Say add some bold for Step 1, Step 2 etc. I think you can also remove the '#' signs.